package.json

```diff
@@ -90,7 +90,7 @@
   "scripts": {
     "build": "node ./node_modules/webpack/bin/webpack.js --mode production",
     "build:dev": "node ./node_modules/webpack/bin/webpack.js --mode development",
-    "codecov": "node ./node_modules/codecov/bin/codecov && rm -rf coverage",
+    "codecov": "node ./node_modules/codecov/bin/codecov --token=995256be-3a1f-4d10-b181-e24f88603b6b && rm -rf coverage",
```

Comment:
We will need to remove all lines about tokens before merging.
I can take care of it before we merge.

Reply:
Okay. I think coverage will work without the token too